components/automate-ui/src/app/modules/infra-proxy/environment-details/environment-details.component.html

```diff
@@ -47,7 +49,7 @@
             <div class="constraint-table">
               <div class="edit">
                 <chef-button class="action" primary (click)="openEditAttributeModal(environment?.default_attributes, 'Constraints')">
-                  <img alt="Edit" src="/assets/img/content and icon.png" />
+                  <img alt="Edit" src="/assets/img/content-and-icon.png" />
```

Comment:
Author note: consistency -- no other file uses spaces in the name. (Besides, having spaces in a filename is asking for maintenance headaches.)